### PR TITLE
VA Medical Facility component

### PR DESCRIPTION
**Pull request by akainic**
Component for selecting preferred VA medical facility. Still to do: 
- Add actual list of options for VA medical facilities
- Filter list based on state selected previously

### CODE REVIEW

**Comment by rickleegit** [comment]
If you need any facility information/data, please let AlexDot or me know. We have a bunch from the Service and Facility locator project.

**Comment by akainic** [comment]
@rickleegit Good to know, thanks Rick! I think we will need it. I'll email you both.

**Comment by awong-dev** [comment]
Awesome Rick!  FYI, we also have the list from the original form in the XML.  @akainic feel free to pass that particular TODO to me for now.

File: spec/javascripts/health-care/components/personal-information/VaMedicalFacility.spec.jsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+    it('onValueChange must be a function', () => {
+      SkinDeep.shallowRender(
+        <VaMedicalFacility onValueChange/>);
+      sinon.assert.calledWithMatch(consoleStub, /Invalid prop `onValueChange` of type `boolean` supplied to `VaMedicalFacility`, expected `function`/);
+    });
+  });
+
+  let component = null;
+  beforeEach(() => {
```
Comment (awong-dev) [inline spec/javascripts/health-care/components/personal-information/VaMedicalFacility.spec.jsx]: There's only one use of this component. Don't use a beforeEach() block as it breaks test case hermeticity.

**Comment by awong-dev** [comment]
LGTM w/ one comment.

Feel free to merge after fixing.

**Pull request merged**